script/v3/DeployAndUpgradeUsccScriptV3.s.sol

```diff
@@ -0,0 +1,58 @@
+pragma solidity ^0.8.20;
+
+import "forge-std/Script.sol";
+import "openzeppelin-contracts-v4/contracts/proxy/transparent/TransparentUpgradeableProxy.sol";
+import "openzeppelin-contracts-v4/contracts/proxy/transparent/ProxyAdmin.sol";
+import "src/AllowList.sol";
+import "src/USCC.sol";
+
+contract DeployAndUpgradeUsccScriptV3 is Script {
```

Comment:
should we just call this `DeployAndUpgradeUsccScript`, to match how the v3 of SuperstateToken is just called `SuperstateToken`, and preserve diff for active version?

Reply:
I'm just following what was already in the repo. No strong opinion